docs/DATASET_INTEGRATION.md

```
@@ -0,0 +1,721 @@
+pipeline.add_annotator(AhocorasickAnnotatorWrapper(WikidataCountryNER()))
+pipeline.add_annotator(AhocorasickAnnotatorWrapper(WikidataProfessionNER()))
+pipeline.add_annotator(AhocorasickAnnotatorWrapper(WikidataDiseaseNER()))
+
+# Extract
+text = "Dr. Smith, a doctor in Paris, treats patients with arthritis. Dogs are popular in many countries."
+for entity in pipeline.process(text):
+    print(f"{entity.entity_type:15} → {entity.value}")
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)